R/rma-tidiers.R

```diff
@@ -0,0 +1,288 @@
+#' @templateVar class rma
+#' @template title_desc_tidy
+#' 
+#' @param x An `rma` created by the `metafor` package.
```

Comment:
I would link to all the function that create rma objects here. That is, something along the lines of:

> #' @param x An `rma` such as those created by [metafor::function1()], [metafor::function2()] or ....